core/templates/components/concept-card/concept-card.template.html

```diff
@@ -4,6 +4,11 @@
       <angular-html-bind class="protractor-test-concept-card-explanation"
                          html-data="$ctrl.currentConceptCard.getExplanation().html">
       </angular-html-bind>
+      <div ng-if="$ctrl.skillDeletedMessage">
+        <div class="oppia-align-center">
+          <span translate="<[$ctrl.skillDeletedMessage]>"></span>
```

Comment:
Would it be sufficient to do `<[$ctrl.skillDeletedMessage]>`? We generally use `translate` attribute when we have translations defined in the language specific JSON files.

Reply:
done PTAL.